### ISSUE TITLE
[Accessbility]A11y_Java Tooling for IntelliJ/Eclipse - New Feature_Application Graph_Screen Reader: Narrator/NVDA is not announcing any information about the controls in 'Application Graph' Section 

### ISSUE THREAD

**Issue by Manisha370** [microsoft/azure-tools-for-java#3628]
[Check out Accessibility Insights!](https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Faccessibilityinsights.io%2F&data=02%7C01%7Cv-tagott%40microsoft.com%7Cb67b2c4b646d4f9561a208d6f4b5c39b%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C636965458847261333&sdata=PJMiKjb64uwhBN98) - Identify accessibility bugs before check-in and make bug fixing faster and easier

GitHubTags: #A11yTCS; #A11yMAS; #DesktopApp; #Win10; #A11y_JavaToolingforIntelliJ/Eclipse-NewFeature; #MAS1.3.1; #ScreenReader; #C+E-H; #JavatoolingforIntelliJ/Eclipse; #Narrator; #NVDA

**Environment Details:**
Eclipse IDE for Enterprise Java Developers
Version: 2019-06 (4.12.0)
Build id: 20190614-1200
Solution Installed: The Azure Toolkit for Eclipse 3.17.0

**OS Details:**
OS: Windows 10 Enterprise (Version 1903, Build: 18362.356)
Screen Readers: Narrator, NVDA
MAS Violated: MAS1.3.1

**Note:**
**Similar issue is observed for the 'Spark Job View' - On pressing "T", Narrator/NVDA is announcing any information about the table details**

**Repro Steps:**
1.Turn on Screen Reader
2.Open Java EE - Eclipse application in light theme
3.Install 'Azure Toolkit for Eclipse' from Marketplace
4.Open Eclipse and install  the Scala plug-in 
5.Tab till tools-->Azure-->Azure Sign in and sign into Azure account and select any subscription
6.In Azure Explorer section, tab till HDInsight and expand it. Select 'Spark on HDInsight (Scala)'. Tab till Next and press enter
7.Tab till Next button and press enter. Press Finish button
8.In Project Explorer, Expand the created Scala project right-click src, New-->Other and press enter
9.Tab till Scala Wizards, expand and select 'Scala Object' and press next
10.Enter the name of the file and press Finish button
11.Open Scala file and write any code in the editor
12.In Project Explorer view, right-click your project, choose Azure-->Submit Spark Application to HDInsight
13.Enter all the required details and press submit button
14.In Azure Explorer tab, tab till HDInsight-->Cluster-->Job and press enter
15.Select any one the application and press enter
16.Verify whether screen reader is reading the information about the controls in the 'Application Graph' section (when we select with mouse and navigating to the controls using Keyboard)

**Actual Result:**
Narrator/NVDA is not announcing any information about the controls in the 'Application Graph' section when keyboard focus is going to the controls

**Expected Result:**
Narrator/NVDA should announce the information about the controls in the 'Application Graph' section when navigating to the controls using 'Tab' key

**User Impact**:
Screen Reader users will not understand about the controls where keyboard focus is going, if it doesn't announce any information

**MAS Reference:**
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={54f28d1f-a2d1-4dcd-84e1-5c9b87e8aba4}

[Bug9_Narrator is not announcing any information about the presence of controls.pptx](https://github.com/microsoft/azure-tools-for-java/files/3651878/Bug9_Narrator.is.not.announcing.any.information.about.the.presence.of.controls.pptx)
![Bug9_NVDA is not announcing any information about the presence of controls when navigation by clicking the section with mouse](https://user-images.githubusercontent.com/51435937/65592275-e5d73a00-dfab-11e9-96eb-0005704c87bc.png)

**Comment by t-rufang** [comment]
The job view page is wrote as static web pages with the help of JavaFX WebView which has no support for Accessibility. Please refer to JDK issue [JDK-8177382](https://bugs.openjdk.java.net/browse/JDK-8177382)

**Comment by Alexandria-R** [comment]
This has been approved to close by SME.

**Comment by vscjavaci** [comment]
Track internal issue by AB#1749733
